### ISSUE TITLE
Error loading saved data: mtr_train_data

### ISSUE THREAD

**Issue by spacecake** [Minecraft-Transit-Railway/Minecraft-Transit-Railway#824]
**Expected Behaviour**
To keep/save on server restart the railroad, trains, depo and elevators.

**Actual Behaviour**
There are all gone. Expected 20-ish train lines and metros

**Reproduction Steps**
Steps to reproduce the behaviour:  created whatever MTR objects on the next server restart is not there.

1.

**Crash Report**
No crash only this error in the logs:
` [Server thread/ERROR] [net.minecraft.world.storage.DimensionSavedDataManager/]: Error loading saved data: mtr_train_data`

**Environment**
Server OS is MineOS.

- Minecraft Version: 1.16.5
- Installed Mods: many-many. 

**Additional Information**
the MTR folder is in the folder and is 775, writable by user/server process. the server and mods where running for atleast 10months (with restarts ).
There was no change in mods or textures from one day to another with the crash, 
There is enough free space and memory.

**Comment by Ultimate-USA-YT** [comment]
Check if MTR Data Saves on a Singleplayer World, than Contact your Host (Excluding Self Host), as they may not be Saving the MTR Folder

**Comment by Ultimate-USA-YT** [comment]
Oh Wait, MineOS is Self-Host. If it's made by Others, Contact Them.

**Comment by spacecake** [comment]
The Minecraft server is 2+ years old, the MTR was added 6+ months ago. With no issues until now. The server was restarted *countless* times.

Anyway, even with the fixes I mentioned above, I believe there is no issue with the MineOS itself. You might me asking why is that?
Because when I remove the MTR folder from the savegame, then suddenly is saves the assets (with all the config/setup/permissions as before for minecraft/mineos).

**Pro**:  New MTR items are saved again
**Contra**: Loss of all previous MTR active items.

On a busy server where they where 50+ lines, its quite an impact to redo.
The issue is not resolved. its a bug. "Who is to blame", I don't know precisely , but somehow I do not think that MineOS has lot to do with this issue..

my two cents for future users having similar issues. 

Thanks to @Ultimate-USA-YT  for trying.

**Comment by BobFish** [comment]
I'm having the same issue on both singleplayer and multiplayer (LAN). I've added the mod to Better MC 1.20 on curse forge https://www.curseforge.com/minecraft/modpacks/better-mc-forge-bmc4 and it seemed to all be working and I spent a few hours yesterday building trains. Unfortunately on a reload all the trains, elevators and track is gone.

Happy to provide any logs or anything if you'd like. Love the mod, hope to be able to use :)
